src/earn/prepareTransactions.test.ts

```diff
@@ -132,91 +130,139 @@ describe('prepareTransactions', () => {
   describe('prepareWithdrawAndClaimTransactions', () => {
     beforeEach(() => {
       jest.mocked(encodeFunctionData).mockReturnValue('0xencodedData')
+      jest.mocked(triggerShortcutRequest).mockResolvedValueOnce({
+        transactions: [
+          {
+            from: '0x123',
+            to: '0x567',
+            data: '0xencodedData',
+            gas: '50200',
+            estimatedGasUse: '49900',
+          },
+        ],
+      })
+      jest.mocked(triggerShortcutRequest).mockResolvedValueOnce({
```

Comment:
nit: do you need the `jest.mocked(triggerShortcutRequest)` again or can the `.mockResolvedValueOnce` just follow after the last one above?

Reply:
+1, would just override the previous mock right?

Reply:
I definitely meant to chain the mockResolvedValueOnce calls, but apparently it has the same effect either way!
I'll change it to chaining so that its more clear